package.json

```diff
@@ -3,11 +3,18 @@
   "license": "MIT",
   "private": true,
   "version": "1.0.0",
-  "main": "./src/index.ts",
-  "types": "./src/types.ts",
+  "main": "./dist/index",
+  "types": "./src/types",
+  "files": [
+    "dist",
+    "src",
+    "data",
+    ".prettierrc"
+  ],
   "scripts": {
     "create-boilerplate": "ts-node src/create-boilerplate.ts",
-    "build": "tsc -p tsconfig.json",
+    "build": "tsc -p tsconfig.json; yarn copy-dts;",
+    "copy-dts": "copyfiles -u 1 \"src/**/*.d.ts\" build",
```

Comment:
Why you need to do this? TS will generate the TS files in the target destination and there are no dts files in src dir.

Reply:
Thanks, you're absolutely right. There was a mistake in the `tsconfig.json`, which I fixed, but then I left this in by accident, which was a fix in itself. I'll remove it. Thanks for the catch!